recipes/stopt/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "stopt" %}
+{% set version = "5.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://gitlab.com/stochastic-control/StOpt
```

Comment:
Please use a tarball here. Github automatically provides those for every tag

Reply:
Done

Reply:
Apologies, a commit has by mistake overwritten the change; I'll restore it